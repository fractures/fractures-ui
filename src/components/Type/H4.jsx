import PropTypes from 'prop-types'
import styled from 'styled-components'

const propTypes = {
	children: PropTypes.node.isRequired,
	className: PropTypes.string
}

const H4 = styled.h4`
	margin-bottom: 0;
	margin-top: 0;

	font-size: 1.25rem;
	line-height: 1.5625rem;
`

H4.propTypes = propTypes

export default H4
