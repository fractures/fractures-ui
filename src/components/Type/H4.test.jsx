import H4 from './H4'

describe('H4', () => {
	const tree = renderer.create(<H4>Extra header</H4>).toJSON()

	it('Matches snapshot', () => {
		expect(tree).toMatchSnapshot()
	})
})
