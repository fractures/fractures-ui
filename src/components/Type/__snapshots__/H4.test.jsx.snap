// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H4 Matches snapshot 1`] = `
<h4
  className="sc-bdVaJa bMIWYQ"
>
  Extra header
</h4>
`;
